src/components/TopBar/CallTime.vue

```diff
@@ -143,6 +150,10 @@ export default {
 			return hours + ' : ' + minutes + ' : ' + seconds
 		},
 
+		showOneHourHint() {
+			return Math.floor((this.callTime / (1000 * 60 * 60)) % 24) >= 1
```

Comment:
Don't know how good it is performance-wise
This value should be re-calculated every time `this.callTime` is updated, so every second?

I'd experiment here a bit, if it's posiible in `mounted` hook get a diff between 1 hour and currently elapsed time, ans use `setTimeout` with this amount of time